### PR TITLE
Bump v0.25.0-2

### PR DESCRIPTION
**Pull request by arg0d**
Update uniffi-bindgen-cpp to v0.2.0+v0.25.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ shows which versions of each generator are inside the docker image.
 
 | Docker image           | uniffi-rs version     | uniffi-bindgen-cs version | uniffi-bindgen-go version | uniffi-bindgen-cpp version |
 |------------------------|-----------------------|---------------------------|---------------------------|----------------------------|
+| v0.25.0-2              | v0.3.0+0.25.0         | v0.7.0+v0.25.0            | v0.2.0+v0.25.0            | **v0.2.0+v0.25.0**         |
```
Comment (Lipt0nas) [inline README.md:42]:
```suggestion
| v0.25.0-2              | v0.3.0+0.25.0         | v0.7.0+v0.25.0            | v0.2.0+v0.25.0            | **v0.2.2+v0.25.0**         |
```

Comment (Lipt0nas) [reply README.md:42]: or do you only write the first major version?

Comment (arg0d) [reply README.md:42]: oops :D

**Review by Lipt0nas — APPROVED** [review]
Nice!

**Pull request merged**